src/components/messenger/list/index.tsx

```diff
@@ -210,6 +221,14 @@ export class Container extends React.Component<Properties, State> {
     );
   };
 
+  renderSecureBackupDialog = (): JSX.Element => {
```

Comment:
Where is the backup dialog currently rendered when you select it from the profile menu? Probably makes sense to remove it from there and just trigger this flag so that it's only rendered in one spot.

That begs the question, is this the right spot?

Anyway, those can be dealt with in follow up PRs but should be considered.